modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -404,6 +438,8 @@ def deserialize(obj):
         else:
             return obj
 
+    logger = get_logger()
+    logger.debug(f"BEGIN::apply_list_of_funcs::{oid_hash}")
```

Comment:
I have tried running some sample scripts (with log level 'all' to get this line to log, but to no avail, despite being able to get the line `logger.debug(f"SUBMIT::_apply_list_of_funcs::{self._identity}")` on line partition.py::116 to log.

Do you have an example script that can log this line? This line would be nice to have because it could help us see start/end of remote computation.

Reply:
I dig some digging and found that the line `logger = get_logger()` returns a logger with level set to WARNING, even when LogMode is set to all.

This is because in [logging/config](https://github.com/modin-project/modin/pull/4372/files#diff-91f7f65eaa299999008d45ca833ad96f6ba57131b0cec49d3df441500617633aR98) logging is configured but LogMode is not set. I'm not sure why this is, since in my debugging script I set `LogMode.put("all")` and in the same script, `Partition.apply` is able to `get_logger` with the right level. I wonder if the envvars is not being picked up by Ray remote functions for some reason.

Reply:
I think the reason for this is that LogMode does not support the `all` flag. As it's currently written it only supports `enable`, `disable`, and `enable_api_only`. If you do `LogMode.put(enable)` you won't have this issue.

Reply:
But you bring up a good point that perhaps I should throw an error of some kind if any other flag is passed

Reply:
I think that you are correct w.r.t. to the ray remotes, will dig into this further.